test-utils/src/index.js

```diff
@@ -0,0 +1,26 @@
+import { Component, options } from 'preact';
+
+/**
+ * Setup a rerender function that will drain the queue of pending renders
+ * @returns {() => void}
+ */
+export function setupRerender() {
+	Component.__test__previousDebounce = options.debounceRendering;
```

Comment:
I know this came from the existing test harness, but the naming is my fault and it's probably worth correcting now haha:

```js
export function setupRerender() {
	const prev = options.debounceRendering;
	options.debounceRendering = cb => prev = cb;
	return () => prev && prev();
}
```

Reply:
Hmm, this does not seem to work, I'll check what's up

Reply:
Doing this change makes me have to change the following test:

```	it('should restore options.debounceRendering after act', () => {
		const spy = sinon.spy();

		options.debounceRendering = spy;
		act(() => null);

		expect(options.debounceRendering).to.equal(spy);
		expect(spy).to.not.be.called;
	});```

To one where spy would get called

Reply:
@developit We did support component-level scheduling for a good while in ceviche before settling for `options.debounceRendering`. Back then you could attach a different scheduler on `Component.debounce`. That's why the test harness still stores the old scheduler function on the component instance.

Reply:
Just to note, the old one `setupRerender` is still in there. I was planning to remove that in another PR for the sake of keeping this one compact. Can still start replacing it in every test file if you prefer.

Reply:
You made the right call, better to do that in another PR 👍